### PR TITLE
Alerting: Fix remaining a11y issues

### PR DESCRIPTION
**Pull request by peterholmberg**
**What this PR does / why we need it**:
Fixes the remaining a11y issues on alerting pages.

**Which issue(s) this PR fixes**:

Part of #40505

**Special notes for your reviewer**:

### CODE REVIEW

File: public/sass/_variables.dark.generated.scss [hunk 1]
```diff
@@ -102,7 +102,7 @@ $text-color: rgb(204, 204, 220);
 $text-color-semi-weak: rgba(204, 204, 220, 0.65);
 $text-color-weak: rgba(204, 204, 220, 0.65);
 $text-color-faint: rgba(204, 204, 220, 0.6);
-$text-color-emphasis: #fff;
+$text-color-emphasis: #FFFFFF;
```
Comment (gillesdemey) [inline public/sass/_variables.dark.generated.scss:105]: Not sure how there got in here 🤔

File: yarn.lock [hunk 2]
```diff
@@ -34235,7 +34235,7 @@ __metadata:
 
 "typescript@patch:typescript@4.3.4#~builtin<compat/typescript>":
   version: 4.3.4
-  resolution: "typescript@patch:typescript@npm%3A4.3.4#~builtin<compat/typescript>::version=4.3.4&hash=493e53"
+  resolution: "typescript@patch:typescript@npm%3A4.3.4#~builtin<compat/typescript>::version=4.3.4&hash=ddd1e8"
```
Comment (gillesdemey) [inline yarn.lock:34238]: Curious why these hashes were updated

**Review by gillesdemey — APPROVED** [review]
LGTM

File: yarn.lock (hunk 2 above)
Comment (peterholmberg) [reply yarn.lock:34238]: should be reverted

File: public/sass/_variables.dark.generated.scss (hunk 1 above)
Comment (peterholmberg) [reply public/sass/_variables.dark.generated.scss:105]: should be reverted

**Pull request merged**